package.json

```diff
@@ -1058,8 +1059,9 @@
         "underscore": "^1.8.3",
         "vscode-azureextensionui": "^0.36.0",
         "vscode-json-languageservice": "^3.0.8",
-        "vscode-languageclient": "^4.4.0",
-        "vscode-languageserver": "^4.4.0",
+        "vscode-languageclient": "^6.1.3",
```

Comment:
Double check the minimum version of VS Code required for these and make sure ours matches it

Reply:
Thank you! Checked the versions it looks good. 
`vscode-languageclient@6.1.3` requires ^1.41.0 version of VSCode engine, current version of VSCode is at 1.48.0
https://github.com/microsoft/vscode-languageserver-node/commit/20a60b0efb23516dd6c4edfedb72d8729c85cdae